raibove/Task4/my_timer.js

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
```suggestion
"use strict";
```
[Check this out](https://javascript.info/strict-mode) to find out why it is recommended to use strict mode.